`azurerm_bot_channel_email` - support for `magic_code` (#23129)
hashicorp/terraform-provider-azurerm: website/docs/r/bot_channel_email.html.markdown

* `email_address` - (Required) The email address that the Bot will authenticate with.

* `email_password` - (Optional) The email password that the Bot will authenticate with.

> * `magic_code` - (Optional) The magic code for setting up the modern authentication(OAUTH).

Reviewer: ```suggestion
* `magic_code` - (Optional) The magic code used to set up OAUTH authentication.
```
